i18n/ja.json

```diff
@@ -320,7 +320,7 @@
     "Preserve your applications, your data, and your sanity.": "アプリケーション、データ、および健全性を維持します。",
-    "CloudLinux's core product, the CloudLinux OS, is a RHEL® fork that has been in place for over ten years. More than 4,000 companies including Dell, Liquid Web, and 1&amp;1 rely on the CloudLinux OS across more than 200,000 product installations. CloudLinux has proven experience in creating and maintaining a RHEL fork and have done so starting with RHEL release 5, right through to release 8. AlmaLinux OS is an opportunity for us to channel our expertise in RHEL into a Linux distribution that serves the broader community. Furthermore, we are including the Linux community right from the inception of AlmaLinux OS. Moving forward, community members will be on the governing board for the AlmaLinux OS project and involved in key decisions too. Finally, AlmaLinux OS will always be free and open source. The community can pick up and continue to develop AlmaLinux OS at any time.": "CloudLinux のコア製品である CloudLinux OS は、10 年以上にわたって導入されてきた RHEL® のフォークです。 Dell、Liquid Web、1&amp;1 を含む 4,000 を超える企業が、200,000 を超える製品インストールで CloudLinux OS に依存しています。 CloudLinux は、RHEL フォークの作成と維持の実績があり、RHEL のリリース 5 からリリース 8 までそうしてきました。AlmaLinux OS は、RHEL に関する私たちの専門知識を、より広範なコミュニティにサービスを提供する Linux ディストリビューションに向ける機会です。さらに、AlmaLinux OS の開始時から Linux コミュニティを巻き込んでいます。今後、コミュニティの参加者は AlmaLinux OS プロジェクトの運営委員会に参加し、重要な決定にも関与することになります。最後に、AlmaLinux OS は常に無料でなおかつオープンソースです。コミュニティはいつでも AlmaLinux OS をフォークして開発を続けることができます。",
+    "CloudLinux's core product, the CloudLinux OS, is a RHEL® fork that has been in place for over ten years. More than 4,000 companies including Dell, Liquid Web, and 1&1 rely on the CloudLinux OS across more than 200,000 product installations. CloudLinux has proven experience in creating and maintaining a RHEL fork and have done so starting with RHEL release 5, right through to release 8. AlmaLinux OS is an opportunity for us to channel our expertise in RHEL into a Linux distribution that serves the broader community. Furthermore, we are including the Linux community right from the inception of AlmaLinux OS. Moving forward, community members will be on the governing board for the AlmaLinux OS project and involved in key decisions too. Finally, AlmaLinux OS will always be free and open source. The community can pick up and continue to develop AlmaLinux OS at any time.": "CloudLinux のコア製品である CloudLinux OS は、10 年以上にわたって導入されてきた RHEL® のフォークです。 Dell、Liquid Web、1&amp;1 を含む 4,000 を超える企業が、200,000 を超える製品インストールで CloudLinux OS に依存しています。 CloudLinux は、RHEL フォークの作成と維持の実績があり、RHEL のリリース 5 からリリース 8 までそうしてきました。AlmaLinux OS は、RHEL に関する私たちの専門知識を、より広範なコミュニティにサービスを提供する Linux ディストリビューションに向ける機会です。さらに、AlmaLinux OS の開始時から Linux コミュニティを巻き込んでいます。今後、コミュニティの参加者は AlmaLinux OS プロジェクトの運営委員会に参加し、重要な決定にも関与することになります。最後に、AlmaLinux OS は常に無料でなおかつオープンソースです。コミュニティはいつでも AlmaLinux OS をフォークして開発を続けることができます。",
```

Comment:
```suggestion
    "CloudLinux's core product, the CloudLinux OS, is a RHEL® fork that has been in place for over ten years. More than 4,000 companies including Dell, Liquid Web, and 1&1 rely on the CloudLinux OS across more than 200,000 product installations. CloudLinux has proven experience in creating and maintaining a RHEL fork and have done so starting with RHEL release 5, right through to release 8. AlmaLinux OS is an opportunity for us to channel our expertise in RHEL into a Linux distribution that serves the broader community. Furthermore, we are including the Linux community right from the inception of AlmaLinux OS. Moving forward, community members will be on the governing board for the AlmaLinux OS project and involved in key decisions too. Finally, AlmaLinux OS will always be free and open source. The community can pick up and continue to develop AlmaLinux OS at any time.": "CloudLinux のコア製品である CloudLinux OS は、10 年以上にわたって導入されてきた RHEL® のフォークです。 Dell、Liquid Web、1&1 を含む 4,000 を超える企業が、200,000 を超える製品インストールで CloudLinux OS に依存しています。 CloudLinux は、RHEL フォークの作成と維持の実績があり、RHEL のリリース 5 からリリース 8 までそうしてきました。AlmaLinux OS は、RHEL に関する私たちの専門知識を、より広範なコミュニティにサービスを提供する Linux ディストリビューションに向ける機会です。さらに、AlmaLinux OS の開始時から Linux コミュニティを巻き込んでいます。今後、コミュニティの参加者は AlmaLinux OS プロジェクトの運営委員会に参加し、重要な決定にも関与することになります。最後に、AlmaLinux OS は常に無料でなおかつオープンソースです。コミュニティはいつでも AlmaLinux OS をフォークして開発を続けることができます。",
```

Reply:
Since my suggestions don't actually change anything you're submitting and are directly complementary, I'm gonna go ahead and merge this in!